ext/openssl/ossl_x509cert.c

```diff
@@ -704,6 +704,84 @@ ossl_x509_eq(VALUE self, VALUE other)
     return !X509_cmp(a, b) ? Qtrue : Qfalse;
 }
 
+static VALUE
+load_chained_certificates(VALUE _io) {
+    BIO *in = (BIO*)_io;
+    VALUE certificates = rb_ary_new();
+
+    X509 *x509 = NULL;
+
+    // Check the certificate format is PEM or DER:
```

Comment:
Please use `/* */` only for now. Ruby <= 2.6 is still pre-C99.

Reply:
Thanks, done.